README.md

```diff
@@ -1,14 +1,18 @@
+# GoldenDict-ng
+
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/e507f9bf83bd48f7a5b76f71dfe9f0dd)](https://app.codacy.com/gh/xiaoyifang/goldendict?utm_source=github.com&utm_medium=referral&utm_content=xiaoyifang/goldendict&utm_campaign=Badge_Grade_Settings)
 [![Windows](https://github.com/xiaoyifang/goldendict/actions/workflows/windows.yml/badge.svg)](https://github.com/xiaoyifang/goldendict/actions/workflows/windows.yml) [![Ubuntu](https://github.com/xiaoyifang/goldendict/actions/workflows/ubuntu.yml/badge.svg)](https://github.com/xiaoyifang/goldendict/actions/workflows/ubuntu.yml)
 [![macos](https://github.com/xiaoyifang/goldendict/actions/workflows/macos.yml/badge.svg)](https://github.com/xiaoyifang/goldendict/actions/workflows/macos.yml)
 
-# GoldenDict-ng
+The Next Generation GoldenDict. A feature-rich open-source dictionary lookup program,
+supporting [multiple dictionary formats](https://xiaoyifang.github.io/goldendict-ng/dictformats/) and online
+dictionaries.
 
-GoldenDict the next generation.
+| Linux | Windows| macOS|
+|---|--|--|
+| [<img src="website/docs/img/linux_genshin.webp" width="500"/>](https://raw.githubusercontent.com/shenlebantongying/goldendict-ng/work/docs_update/website/docs/img/linux_genshin.webp) | [<img src="website/docs/img/windows_white.webp" width="500"/>](https://raw.githubusercontent.com/shenlebantongying/goldendict-ng/work/docs_update/website/docs/img/windows_white.webp) | [<img src="website/docs/img/mac_black.webp" width="500"/>](https://raw.githubusercontent.com/shenlebantongying/goldendict-ng/work/docs_update/website/docs/img/mac_black.webp) |
```

Comment:
link

Reply:
They are links with images as content,
```
 v html img
[<img src=“”>](https://link_url)
^markdown link
```

Reply:
Can these `https://raw.githubusercontent.com/shenlebantongying` links use the host of this fork

Reply:
:sweat_smile: copy paste error

Reply:
fixed